construction/static/description/index.html

```diff
@@ -4,4 +4,9 @@ <h2><strong>Tailored Setup for Construction Businesses</strong></h2>
     focusing on accurate quoting, efficient planning, seamless execution, and excellent customer service.
     The setup integrates all necessary modules (Project, Inventory, Sales, Purchase, ...) to streamline your operations,
     with the possibility to expand into advanced functionalities using additional applications.
+
+    <h3>Customizations</h3>
+    <ul>
+        <li><strong>Progress management</strong> – Easily handle sale orders and project progress in a dedicated view</li>
```

Comment:
Could you move the diff from construction to the next PR where we add the setting?